src/test/java/org/littleshoot/proxy/KeepAliveTest.java

```diff
@@ -85,7 +85,7 @@ public void testHttp11DoesNotCloseConnectionByDefault() throws IOException, Inte
         this.socket = SocketClientUtil.getSocketToProxyServer(proxyServer);
 
         // construct the basic request: METHOD + URI + HTTP version + CRLF (to indicate the end of the request)
-        String successfulGet = "GET http://localhost:" + mockServerPort + "/success HTTP/1.1\n"
```

Comment:
Fixed the requests to send CRLF CRLF at the end of request throughout whole class.